src/maintenance/setup.py

```diff
@@ -10,7 +10,7 @@
 from setuptools import setup, find_packages
 
 # HISTORY.rst entry.
-VERSION = '1.6.0'
+VERSION = 1.7.0b1'
```

Comment:
Missing single quote here
```suggestion
VERSION = '1.7.0b1'
```